components/HomeScreen/Title/Title.utils.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @callback setGlobeOrientation
+ * @param {number}
+ */
+
+/**
+ * yieldRotateGlobe.
+ *
+ * @param {number} globeOrientation
+ * @param {function} setGlobeOrientation
+ */
+export function yieldRotateGlobe(globeOrientation, setGlobeOrientation) {
```

Comment:
you could start using object params on your functions ({globeOrientation, setGlobeOrientation}) for the various reasons we talked about